sdk/purview/Azure.Data.Purview.Scanning/src/autorest.md

```diff
@@ -0,0 +1,26 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+title: Scanning
+input-file: ../../../../../azure-rest-api-specs-pr/specification/purview/data-plane/Microsoft.Purview/preview/2018-12-01-preview/scanningService.json
```

Comment:
The source swagger has not landed in azure-rest-api-specs yet, so this points to a local copy on my machine.  I suspect this will have to change before we can land this (because I recall we have automation that tries to regenerate the code and this will fail to do so on everyone else's machine).  We are working with the service team to get a public swagger (at least as a Pull Request) that we can point at.

Reply:
Yes, we have to check in the swagger either locally or point to a PR for automation to be happy.